lib/spack/docs/getting_started.rst

```diff
@@ -885,51 +894,59 @@ Environment Modules
 In order to use Spack's generated environment modules, you must have
 installed one of *Environment Modules* or *Lmod*.  On many Linux
 distributions, this can be installed from the vendor's repository.  For
-example: ``yum install environment-modules`` (Fedora/RHEL/CentOS).  If
-your Linux distribution does not have Environment Modules, you can get it
-with Spack:
+example: ``yum install environment-modules`` (Fedora/RHEL/CentOS). If
+your Linux distribution does not have Environment Modules, Spack can
+build it for you!
 
-#. Consider using system tcl (as long as your system has Tcl version 8.0 or later):
+What follows are three steps describing how to install and use environment-modules with spack.
```

Comment:
This is two steps now isn't it (with your most recent changes)?

Reply:
The third step is to test that the module command works properly.
Do you think I should do away with that step?

Reply:
Sorry I missed that - nope I see 3 and I like the 3rd step for verification.